src/fakeData/fakeHealthcareProfessionals.ts

```diff
@@ -15,6 +15,24 @@ export function generateRandomCreateHealthcareProfessionalInput(
     }
 }
 
+export function generateFailingName(
+    { facilityIds }: { facilityIds: string[] } = { facilityIds: [] }
```

Comment:
I'm not quite sure where to put this logic for creating different tests for testing the validations. Since there are multiple validations per field, I'm not sure what the best approach is.